sources/react/template-devdocs-example.js

```diff
@@ -0,0 +1,30 @@
+const prepareDevDocsExample = components => `
+/* eslint-disable no-alert */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from './index.js';
```

Comment:
Can we actually just drop the `.js` extension?

Reply:
By leaving the extension we make sure it works in any native environment (and doesn't depend on a bundler's ability to work without the extension, etc).

Reply:
Hmm. Sure about this? Wouldn't any native environment that supports ES Modules also be able to resolve this without the extension? After all, there are a number of extension-free `import`s, aren't there?

Reply:
What I [read](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) is that browsers won't understand that without the extension.

Reply:
I'm happy to change it if it makes things clear, though.

Reply:
Done at d2d1478